TheThingsNetwork/lorawan-stack
# Resolve small issues after message deduplication

#### Summary
This quickfix will add a few missed messages as well as bring back Japanese translations of shared messages.

#### Changes

- Use correct message ID for `Admin panel`
- Move `MAC data` message to shared messages for better consistency with enterprise
- Reintroduce Japanese shared messages

#### Testing

1. Navigate to Admin Panel
2. See the message "Admin panel" in the breadcrumbs at the top of the page

#### Checklist
